frontend/src/pages/Notifications/NotificationPageContainer.js

```diff
@@ -31,9 +38,7 @@ class NotificationPageContainer extends Component {
 
   render() {
     return (
-      <div style={globalStyles.innerContainer}>
-        <NotificationPage {...this.props} />
-      </div>
+      <div style={globalStyles.innerContainer}>{!this.state.data ? <div /> : <NotificationPage {...this.props} />}</div>
```

Comment:
why don't we use `isDataLoading` here?

Reply:
It's used in NotificationList.js, so that while loading the background and the head of the table is visible